xocto/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "4.5.0"
+__version__ = "4.5.1"
```

Comment:
1. Not a PATCH update - what bug does this fix?
2. Don't mix code changes with version updates

Reply:
If the user had a newer version of `pandas` then using the local filestore version of `select_object_contents` would error. Would that count as a bug?

Reply:
I've bumped this to 4.6.0 for now 👍